pkg/messaging/v1/util.go

```diff
@@ -65,6 +65,9 @@ const (
 	errorInfoDomain            = "dapr.io"
 	errorInfoHTTPCodeMetadata  = "http.code"
 	errorInfoHTTPErrorMetadata = "http.error_message"
+
+	CallerIDHeader = "caller-app-id"
+	CalleeIDHeader = "callee-app-id"
```

Comment:
I think these two headers should be prefixed by `DaprHeaderPrefix` from line 48

Reply:
Just added the prefix